ezyrb/matlabhandler.py

```diff
@@ -1,67 +1,63 @@
 """
 Derived module from filehandler.py to handle Matlab files.
 """
+import os
 import numpy as np
 import scipy.io as sio
-import ezyrb.filehandler as fh
 
 
-class MatlabHandler(fh.FileHandler):
+class MatlabHandler(object):
 	"""
-	Matlab format file handler class
+	Matlab format file handler class.
+	You are NOT supposed to call directly this class constructor (use
+			Filehandler constructor instead)
 
-	:cvar string infile: name of the input file to be processed.
-	:cvar string extension: extension of the input/output files. It is equal to '.mat'.
-	:cvar string output_name: name of the output of interest inside the mat file. Default value is output.
+	:cvar private _filename: name of file to handler
```

Comment:
With private we are missing the type of the variable. I have not see the result of the compiled doc yet. Perhaps using string after cvar and then describing the member as private in the description would solve the issue.